jenkins/model_output_processing_Jenkinsfile

```diff
@@ -74,21 +74,6 @@ pipeline {
           aws s3 cp date.txt s3://prod-owi-resources/resources/Application/wbeep/${TIER}/date.txt
           aws s3 cp . s3://wbeep-${TIER}-website/estimated-availability/date/test_results/ --recursive --exclude "*" --include "*${DATE}.csv" --include "order_of_magnitude_test_${DATE}.txt"
         '''
-        script {
```

Comment:
i may add this back after more discussion, but right now it feels redundant to the extensive data checks we are doing